front_end/panels/ai_chat/ui/settings/providers/OpenRouterSettings.ts

```
@@ -0,0 +1,537 @@
+      .oauth-status {
+        font-size: 12px;
+        margin-top: 5px;
+        padding: 5px 8px;
+        border-radius: 4px;
+        background: var(--color-background-highlight);
+      }
+    `;
+    document.head.appendChild(oauthStyles);
+
+    // Update OAuth button state
+    const updateOAuthButton = async () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)